pages/currency-exchange/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import personData from 'mock/person.json';
+import NavBar from '@components/NavBar';
+import Head from 'next/head';
```

Comment:
NIT: This should be placed at line no 2

Reply:
didn't get exactly get it sumit?

Reply:
this import statement should be coming after line no 1. (good practise)